doc/api/nmo-import-mongo.md

```diff
@@ -0,0 +1,11 @@
+nmo-import-mongo(3) -- import-mongo
+==============================
+
+## SYNOPSIS
+
+    nmo.commands.import-mongo([<url> || <cluster>], [database], [mongourl], [collection])
```

Comment:
most of them the args are required, see https://github.com/apache/couchdb-nmo/pull/16/files#r46426220